common/djangoapps/student/tests/test_admin_views.py

```diff
@@ -2,7 +2,13 @@
 Tests student admin.py
 """
 from django.core.urlresolvers import reverse
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.contrib.admin.sites import AdminSite
 
+from mock import Mock
```

Comment:
NIT: `mock` should be grouped with the other 3rd party dependencies above.  AKA remove line 8